spec/shipcloud/shipment_spec.rb

```diff
@@ -47,28 +47,32 @@
 
   describe ".create" do
     it "makes a new POST request using the correct API endpoint" do
-      Shipcloud.should_receive(:request).with(:post, "shipments", valid_attributes).and_return("data" => {})
+      expect(Shipcloud).to receive(:request).with(:post, "shipments", valid_attributes, api_key: nil).
+        and_return("data" => {})
       Shipcloud::Shipment.create(valid_attributes)
     end
   end
 
   describe ".find" do
     it "makes a new GET request using the correct API endpoint to receive a specific subscription" do
-      Shipcloud.should_receive(:request).with(:get, "shipments/123", {}).and_return("id" => "123")
+      expect(Shipcloud).to receive(:request).with(:get, "shipments/123", {}, api_key: nil).
+        and_return("id" => "123")
       Shipcloud::Shipment.find("123")
     end
   end
 
   describe ".update" do
     it "makes a new PUT request using the correct API endpoint" do
-      Shipcloud.should_receive(:request).with(:put, "shipments/123", {:carrier => 'ups' }).and_return("data" => {})
+      expect(Shipcloud).to receive(:request).with(:put, "shipments/123", {:carrier => 'ups' }, api_key: nil).
```

Comment:
Line is too long. [109/100]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.